app/Http/Controllers/ContactController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\DB;
+
+use App\Contact;
+
+class ContactController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $contacts = DB::table('contacts')->get();
+
+        foreach ($contacts as $contact) {
+            try {
+                $response = file_get_contents(env('EMAIL_SUBSCRIPTION_API') . '/get_status?email=' . $contact->email);
```

Comment:
[Bad]- It would have been good to make use of the resource collection here since that is what we are passing to the view.
For more detail take a look at the link below
https://laravel.com/docs/8.x/eloquent-resources#resource-collections